po/de.po

```diff
@@ -0,0 +1,90 @@
+# German translation for gnome-runcat-extension package.
+# Copyright (C) 2023 WebSnke
+# This file is distributed under the same license as the gnome-runcat-extension package.
+# WebSnke <websnke@tutanota.com>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-runcat-extension 21\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-13 16:30+0100\n"
+"PO-Revision-Date: 2023-03-13 16:30+0100\n"
+"Last-Translator: WebSnke\n"
+"Language-Team: none\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/panelMenuButton.js:95
+msgid "Open System Monitor"
+msgstr "Open System Monitor"
+
+#: src/panelMenuButton.js:99
+msgid "Settings"
+msgstr "Settings"
+
+#: src/prefs.js:48
+msgid "RunCat Settings"
+msgstr "RunCat Settings"
+
+#: src/prefs.js:59
+msgid "Version"
+msgstr "Version"
+
+#: src/resources/ui/preferences.ui:6
+msgid "General"
+msgstr "Allgemein"
+
+#: src/resources/ui/preferences.ui:10
+msgid "General Preferences"
+msgstr "Allgemeine Einstellungen"
+
+#: src/resources/ui/preferences.ui:14
+msgid "Idle threshold"
+msgstr "Leerlaufzeitschwellenwert"
+
+#: src/resources/ui/preferences.ui:34
+msgid "Displaying items"
+msgstr "Artikel anzeigen"
```

Comment:
I’d go with `Anzeigen als` here.

This is the header for a list, isn’t it? And that list shows `Character and percentage`, `Percentage only`, and `Character only`, right? So, the user selects how the CPU load is being displayed. (You might even want to call it `Show as` in English, dunno.)

Reply:
[Screencast from 2023-03-14 10-44-26.webm](https://user-images.githubusercontent.com/11278181/224930381-e477086a-c96a-45db-beab-763b0eb71365.webm)

Reply:
Ah, thanks. I just changed it to "Anzeigen als" as @vain suggested.